digitaloceanmodule.php

```diff
@@ -2,25 +2,11 @@
 
 class Digitaloceanmodule extends Module {
 
-    private static $version = "2.2";
-    private static $authors = array(array('name' => "ModulesBakery.com.", 'url' => "http://www.modulesbakery.com"));
```

Comment:
ModulesBakery is the original author, so they should be included in any place authors are listed.